d2l-focusable-arrowkeys-behavior.js

```diff
@@ -128,6 +128,7 @@ D2L.PolymerBehaviors.FocusableArrowKeysBehavior = {
 		var target = e.target;
 		if (this.arrowKeyFocusablesDirection.indexOf('left') >= 0 && e.keyCode === this.__keyCodes.LEFT) {
 			fastdom.measure(function() {
+				if (!this.isConnected) return;
```

Comment:
There were some weird timing-related errors due to the async nature of this code and it running _after_ the tests had moved on and the nodes no longer being connected anymore.